selftests/functional/test_standalone.py

```diff
@@ -62,6 +62,13 @@ def test_failtest_nasty2(self):
         self.assertIn("Exception: Unable to get exception, check the traceback"
                       " for details.", result.stdout)
 
+    def test_failtest_nasty3(self):
```

Comment:
Well, that's true. Anyway as there are 3 of them, I'd prefer to do it as separate PR...